packages/ide-extension/src/panel/guidedAnswersPanel.ts

```diff
@@ -161,7 +162,8 @@ export class GuidedAnswersPanel {
                     break;
                 }
                 case SEND_FEEDBACK_COMMENT: {
-                    await this.guidedAnswerApi.sendFeedbackComment(action.payload);
+                    const commentResponse = await this.guidedAnswerApi.sendFeedbackComment(action.payload);
+                    this.postActionToWebview(feedbackResponse(commentResponse.status === 200));
```

Comment:
In `guided-answers-api.ts -> postFeedback()` we already check `response.status !== 200` and throw an error in case: https://github.com/SAP/guided-answers-extension/pull/302/files#diff-e7dfaf728dc2b100d5c14d898b74936dda2b64ca50f7fd4da200b6ad1b12db61R228

Either we add another try/catch around the call, or we remove the if condition in `postFeedback()`.
My vote goes to remove `throw Exception` and return the status.

Reply:
Done, thanks!